HelperFunctions/PeakFitting/fitpeaks.m

```diff
@@ -203,6 +205,9 @@
 vals.amplitudes = fitcoeffs(1:n);
 vals.locations = fitcoeffs(n+1:2*n);
 vals.widths = fitcoeffs(2*n+1:3*n);
+if strcmp(lower(p.FitType),'voigt')
```

Comment:
`strcmpi` is more efficient that calling `lower` and `strcmp`.

Reply:
Fixed - realized that already called lower(p.FitType) earlier